assets/app/features/upgrade/templates/openstack-services.jade

```diff
@@ -4,7 +4,7 @@ p(translate='') upgrade.steps.openstack-services.description
 button.btn.btn-success.center-block(
     ng-click='upgradeOpenStackServicesVm.openStackServices.stopServices()',
     ng-disabled='upgradeOpenStackServicesVm.checks.services.running || \
-        upgradeOpenStackServicesVm.checks.backup.completed || upgradeOpenStackServicesVm.checks.backup.running',
```

Comment:
I found it while testing the behavior wrt the design spec so it's both actually. But you're right, it should have been included in #134.